### DISCUSSION TITLE
html

### DISCUSSION THREAD

**Discussion by PRANAVSAISRI** [Snailclimb/JavaGuide#2200] [Q&A]
<html>
<body>

<!doctype html>
--
  | <html dir="ltr" lang="en" class="loading"
  | >
  | <head>
  | <meta charset="utf-8">
  | <meta name="color-scheme" content="light dark">
  | <title>Settings</title>
  | <base href="chrome://settings">
  | <link rel="stylesheet" href="chrome://resources/css/md_colors.css">
  | <style>
  | html {
  | overflow: hidden;
  | /* Remove 300ms delay for 'click' event, when using touch interface. */
  | touch-action: manipulation;
  | }
  |  
  | @media (prefers-color-scheme: dark) {
  | html {
  | background: var(--md-background-color);
  | }
  | }
  |  
  | html.loading::before {
  | background-color: var(--md-toolbar-color);
  | border-bottom: var(--md-toolbar-border);
  | box-sizing: border-box;
  | content: '';
  | display: block;
  | height: var(--md-toolbar-height);
  | }
  |  
  | html,
  | body {
  | height: 100%;
  | margin: 0;
  | }
  | </style>
  | </head>
  | <body>
  | <link rel="stylesheet" href="chrome://resources/css/text_defaults_md.css">
  | <script type="module" src="settings.js"></script>
  | <settings-ui></settings-ui>
  | </body>
  | </html

</body>
</html><!doctype html>
<html dir="ltr" lang="en" class="loading"
    >
<head>
  <meta charset="utf-8">
  <meta name="color-scheme" content="light dark">
  <title>Settings</title>
  <base href="chrome://settings/">
  <link rel="stylesheet" href="chrome://resources/css/md_colors.css">
  <style>
    html {
      overflow: hidden;
      /* Remove 300ms delay for 'click' event, when using touch interface. */
      touch-action: manipulation;
    }

    @media (prefers-color-scheme: dark) {
      html {
        background: var(--md-background-color);
      }
    }

    html.loading::before {
      background-color: var(--md-toolbar-color);
      border-bottom: var(--md-toolbar-border);
      box-sizing: border-box;
      content: '';
      display: block;
      height: var(--md-toolbar-height);
    }

    html,
    body {
      height: 100%;
      margin: 0;
    }
  </style>
</head>
<body>
  <link rel="stylesheet" href="chrome://resources/css/text_defaults_md.css">
  <script type="module" src="[settings.js](chrome://settings/settings.js)"></script>
  <settings-ui></settings-ui>
</body>
</html

**Comment by n00byN00byD0** [comment]
⽕ in your butt??

**Reply by 123hilv1hd** [reply]
ye

**Comment by fatilozoya23-eng** [comment]
> ## <!doctype html>
>   | <html dir="ltr" lang="en" class="loading"   | >   |   |   |   | <title>Settings</title>   |   |   | <style>   | html {   | overflow: hidden;   | /* Remove 300ms delay for 'click' event, when using touch interface. */   | touch-action: manipulation;   | }   |     | @media (prefers-color-scheme: dark) {   | html {   | background: var(--md-background-color);   | }   | }   |     | html.loading::before {   | background-color: var(--md-toolbar-color);   | border-bottom: var(--md-toolbar-border);   | box-sizing: border-box;   | content: '';   | display: block;   | height: var(--md-toolbar-height);   | }   |     | html,   | body {   | height: 100%;   | margin: 0;   | }   | </style>   |   |   |   | <script type="module" src="settings.js"></script>   |   |   | </html
> 
> <title>Settings</title> <style> html { overflow: hidden; /* Remove 300ms delay for 'click' event, when using touch interface. */ touch-action: manipulation; }
> ```
> @media (prefers-color-scheme: dark) {
>   html {
>     background: var(--md-background-color);
>   }
> }
> 
> html.loading::before {
>   background-color: var(--md-toolbar-color);
>   border-bottom: var(--md-toolbar-border);
>   box-sizing: border-box;
>   content: '';
>   display: block;
>   height: var(--md-toolbar-height);
> }
> 
> html,
> body {
>   height: 100%;
>   margin: 0;
> }
> ```
> 
> </style> <script type="module" src="[settings.js](chrome://settings/settings.js)"></script>
